File: cv07b/app/src/main/java/com/example/myapp008bfragmentsexample1/MainActivity.java
package com.example.myapp008bfragmentsexample1;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        findViewById(R.id.showFormFragmentButton).setOnClickListener(v ->
                getSupportFragmentManager().beginTransaction()
                        .replace(R.id.fragmentContainer, new Fragment1())
                        .commit()
        );

        findViewById(R.id.showCircleFragmentButton).setOnClickListener(v ->
                getSupportFragmentManager().beginTransaction()
                        .replace(R.id.fragmentContainer, new Fragment2())
                        .commit()
        );
    }
}

File: cv14/app/src/main/java/com/example/myapp016avanocniappka/MainActivity.kt
package com.example.myapp016avanocniappka

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import androidx.recyclerview.widget.GridLayoutManager
import androidx.recyclerview.widget.RecyclerView

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val recyclerView: RecyclerView = findViewById(R.id.recyclerView)

        // Vytvoření seznamu názvů her
        val games = listOf("Pexeso", "Kvíz")

        // Nastavení LayoutManageru pro mřížku
        recyclerView.layoutManager = GridLayoutManager(this, 2) // 2 sloupce

        // Nastavení adapteru pro RecyclerView
        recyclerView.adapter = TileAdapter(this, games)
    }
}
